packages/setup-ocaml/src/windows.ts

```diff
@@ -55,14 +55,12 @@ export async function setupCygwin() {
     }
     const packages = [
       "curl",
-      "diffutils",
       "m4",
       "make",
       "mingw64-i686-gcc-core",
       "mingw64-i686-gcc-g++",
       "mingw64-x86_64-gcc-core",
       "mingw64-x86_64-gcc-g++",
-      "patch",
```

Comment:
diffutils is part of base anyway, also it contains utilities required per POSIX standard such as `find` and `diff` which would be missing otherwise.

Same for `patch`. Even if it is no longer used by opam itself, it is still a [required POSIX utility](https://pubs.opengroup.org/onlinepubs/9799919799/utilities/patch.html)

Reply:
Actually i'm not sure why cygwin base doesn't require `patch`, but maybe it's just because cygwin isn't a POSIX emulation layer but a Linux one and most Linux distributions don't really seem to care about POSIX.

Reply:
It might be worth keeping diffutils, right?

Reply:
I see